Gems/Atom/RPI/Code/Source/RPI.Builders/Material/MaterialBuilder.cpp

```diff
@@ -234,15 +260,15 @@ namespace AZ
             const auto& materialSourceData = materialSourceDataOutcome.GetValue();
 
             // Load the material file and create the MaterialAsset object
-            auto materialAssetOutcome = materialSourceData.CreateMaterialAsset(
+            const auto& materialAssetOutcome = materialSourceData.CreateMaterialAsset(
```

Comment:
`CreateMaterialAsset` does not return a const ref

Reply:
Old habit. It does not return one but avoids copying, which probably is not necessary anymore, by extending the lifetime of a referenced temporary.